pkgs/development/python-modules/TakeTheTime/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "TakeTheTime";
```

```suggestion
  pname = "takethetime";
```
we normalise pname and attrs